_pages/yaml-file-format.md

```diff
@@ -116,6 +116,17 @@ Their names may be changed a YAML file with a `lang` other than `en`.
     
     The list may be incomplete, as a new calendar might be defined that uses an existing month.
 
+-   <table><tbody>
+    <tr><th>Key</th><td><code>change controller</code></td></tr>
+    <tr><th>Type</th><td><code>str</code></td></tr>
+    <tr><th>Required by</th><td>—</td></tr>
+    <tr><th>Allowed by</th><td>all</td></tr>
+    </tbody></table>
+
+    An identifier of who is allowed to make changes to this file.  For example, it could be
```

Comment:
```suggestion
    An identifier of who is allowed to authorize changes to this file.  For example, it could be
```

I worry that "make" could imply being a committer on github, which I don't think that this field is intended to mean.